engine/swarm/secrets.md

```diff
@@ -846,3 +846,80 @@ the information from a Docker-managed secret instead of being passed directly.
 >**Note**: Docker secrets do not set environment variables directly. This was a
 conscious decision, because environment variables can unintentionally be leaked
 between containers (for instance, if you use `--link`).
+
+## Using Docker Secrets in Compose
+
+For this example we will create a simple WordPress site using Docker Secrets in
```

Comment:
When you say `using Docker Secrets`, it seems that you're talking about a product. But this is a feature just like many others so it can be `... using secrets in a Compose file`.